src/wallet/rpcwallet.cpp

```diff
@@ -1363,9 +1363,13 @@ UniValue listtransactions(const JSONRPCRequest& request)
             "[\n"
             "  {\n"
             "    \"address\":\"address\",    (string) The bitcoin address of the transaction.\n"
-            "    \"category\":\"send|receive\", (string) The transaction category.\n"
-            "    \"amount\": x.xxx,          (numeric) The amount in " + CURRENCY_UNIT + ". This is negative for the 'send' category, and is positive\n"
-            "                                        for the 'receive' category,\n"
+            "    \"category\":               (string) The transaction category.\n"
+            "                \"send\"|                 'send' specifies transactions sent.\n"
```

Comment:
I don't see the need to repeat. Something along this?
 ```
 "    \"category\"                    (string) The transaction category. One of\n" 
 "        \"send\"                    transactions sent\n"
 "        \"receive\"                 non-coinbase transations received\n" 
...
```
However I don't have strong opinion here and currently there is no "standard" output defined (that I know of).

Reply:
Reformatted to remove the repetition.